components/org.wso2.carbon.identity.oauth.endpoint/pom.xml

```diff
@@ -82,12 +82,44 @@
             <artifactId>oltu</artifactId>
             <scope>provided</scope>
         </dependency>
+        
+        <!-- CXF related dependency  -->
+        <!-- TODO once all the cxf-bundle related components are upgraded in other repos, we could add  
+          'provided' scope for cxf bundles here and pack them in the lib/cxf/runtime through 
+         carbon.deployment module    
+         -->  
         <dependency>
-            <groupId>org.apache.cxf.wso2</groupId>
-            <artifactId>cxf-bundle</artifactId>
-            <scope>provided</scope>
+	    <groupId>org.apache.cxf</groupId>
+	    <artifactId>cxf-core</artifactId>
```

Comment:
Doesn't this need to be provided ? If this is provided does it bundle anything ? Why do we need to exclude these explicitly ?